src/integrations/contact-input-custom-chip-component.tsx

```diff
@@ -15,6 +15,11 @@ import styled from 'styled-components';
 
 import { ContactInputOnChange, ContactInputValue, CustomChipProps } from '../types/integrations';
 
+export const CHIP_DISPLAY_NAME_VALUES = {
+	LABEL: 'label',
+	EMAIL: 'email'
+};
```

Comment:
Add as const to have a strong typing
```suggestion
} as const;
```